bookserver/models.py

```diff
@@ -143,6 +145,7 @@ def copy(self, **kw):
 
 # The parsons_answers table is the only outlier in that it adds a source column to keep
 # track of which blocks were not used in the answer.
+# TODO: What about timed exams?
```

Comment:
I think the timed exam table is a different beast than the xxx_answers tables.

Reply:
Sounds good. I'll remove this notes.